enos/modules/go-acceptance-test-target/main.tf

```diff
@@ -77,3 +79,27 @@ resource "aws_security_group" "this" {
     cidr_blocks = ["0.0.0.0/0"]
   }
 }
+
+resource "aws_instance" "remotehost" {
+  ami           = data.aws_ami.ubuntu.id
+  instance_type = "t3.small"
+  key_name      = var.ssh_key_name
+
+  vpc_security_group_ids = [aws_security_group.this.id]
+
+  tags = {
+    Name = "enos_provider_remote_host"
+  }
+}
+
+resource "enos_remote_exec" "wait" {
+  inline = ["sudo cloud-init status --wait"]
```

Comment:
I also added this so our tests timeout while waiting for the instance to actually be available.